src/index.js

```diff
@@ -59,9 +65,19 @@ function IpfsAPI (host_or_multiaddr, port) {
 
   // -- Interface
 
+  /** @public */
   self.send = requestAPI
 
-  self.add = (files, opts, cb) => {
+  /**
+   * Add a file/many files to IPFS returning the hash and name. The
+   * name value will only be set if you are actually sending a file.
+   *
+   * @public
+   * @param {} files
+   * @param {object} opts
+   * @param {function} cb
+   */
```

Comment:
How does everyone feel about this JavaDoc'ization in order to have docs? 

It is nice to have API docs, not sure if, since we have to write them anyway, if we prefer to right them on the readme with examples or add this to all of the function calls.